workflow-bot-app/src/test/resources/form/send-form-unique-reply-timeout.swadl.yaml

```diff
@@ -0,0 +1,30 @@
+id: send-form-timeout
+activities:
+  - send-message:
+      id: sendForm
+      content: |
+        <messageML>
+          <form id="sendForm">
+            <button name="approve" type="action">Approve</button>
+            <button name="reject" type="action">Reject</button>
+          </form>
+        </messageML>
+      on:
+        message-received:
+          content: /send
+      to:
+        stream-id: "123"
+
+  - execute-script:
+      id: replyWithTimeout
+      on:
+        timeout: PT0.01S
+        form-replied:
+          form-id: sendForm
+          unique: true
```

Comment:
we should figure out what is the best name for it, unique, reply ? and what should be the default value
it is probably worth discussing with dev rel team

Reply:
I keep and document this waiting for feedback from devRel team